test/unit/connection/connection_config_test.js

```diff
@@ -231,6 +231,56 @@ describe('ConnectionConfig: basic', function ()
           },
         errorCode: ErrorCodes.ERR_CONN_CREATE_INVALID_ACCOUNT
       },
+      {
+        name: 'account with invalid character',
```

Comment:
could we add also some tests with valid account examples?

Reply:
Sure, done.